app/controllers/admin/orders_controller.rb

```diff
@@ -1,4 +1,29 @@
 class Admin::OrdersController < Admin::BaseController
+
   def index
+    # user = User.find_by(username: params[:session][:username])
+    # if user.current_admin
+    # @orders = Order.all
+    @orders = Order.all
+    # byebug
+      # byebug
+    # else
+    #   render:
+    # end
+    @status = Order.group(:status).count
+    byebug
   end
+
+  def sort_orders
+    # status = params[:order][:status]
+      @orders = Order.get_by_status(params[:order][:status])
```

Comment:
when they use the dropdown @orders is reset to only include the orders corresponding to the selected status. get_by_status is a class method in the order model. I think this is something that can be refactored.